Backends/CLX/medium.lisp

```diff
@@ -361,6 +361,21 @@ translated, so they begin at different position than [0,0])."))
 
 ;;; XXX: both PM and MM pixmaps should be freed with (xlib:free-pixmap pixmap)
 ;;; when not used. We do not do that right now.
+(defun put-image-recursively (pixmap pixmap-context pixmap-image width height x0 y0)
+  (cond
+    ((and (< width 2048) (< height 2048))
```

Comment:
Does clx export symbolic constants for these numbers? If yes, we should use those. If not, we should make our own (or add them in clx).

Reply:
As we discussed on IRC, the number two to the eleventh is known to CLX. Let me quote from Backends/CLX/keysyms.lisp:

```
;;; The parameter STATE is a bit mask represented as the logical OR
;;; of individual bits.  Each bit corresponds to a modifier or a
;;; pointer button that is active immediately before the key was
;;; pressed or released.  The bits have the following meaning:
;;;
;;;   position  value    meaning
;;;   [...]
;;;    11      2048      button4
```

As we know from the release notes to System 286 [1], All Number are documented:

```
[5] Number are documented.

    As part of our policy of keeping things well documented, all numbers
    in the system must be documented.  Creation of a number without any
    documentation will result in an error.  Use of the DESCRIBE function
    on a number will print out its documentation.  Note that numbers are
    EQ now, so don't try to use one number for more than one purpose.
    (That's why there's an infinite number of them.) (Well, almost, 
    as we only have a sex bit machine.)
```

[1]: http://www.ibiblio.org/pub/docs/humor/fionavar/sys286_msg